datamations/datamation_groupby.py

```diff
@@ -273,7 +301,7 @@ def prep_specs_summarize(self, width=300, height=300):
         y_encoding = {
             "field": "datamations_y",
             "type": "quantitative",
-            "title": [self.operations[-1] + " of", y_axis] if self.operations[-1] == "median" else self.operations[-1] + "(" + y_axis + ")",
+            "title": [self.operations[-1] + " of", y_axis] if self.operations[-1] == "median" or any(element in ['min', 'max'] for element in self.operations) and len(self._by) > 1 else self.operations[-1] + "(" + y_axis + ")",
```

Comment:
How about:
`any(element in ['median', 'min', 'max'] for element in self.operations`